File: GXpert/GXpert.Web/Modules/ServerTypes/Content/ContentAudioColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { ContentAudioRow } from "./ContentAudioRow";

export interface ContentAudioColumns {
    Id: Column<ContentAudioRow>;
    MediaFile: Column<ContentAudioRow>;
    HlsFile: Column<ContentAudioRow>;
    FileKeyUrl: Column<ContentAudioRow>;
    Iv: Column<ContentAudioRow>;
    ContentTitle: Column<ContentAudioRow>;
    LanguageName: Column<ContentAudioRow>;
    Username: Column<ContentAudioRow>;
}

export class ContentAudioColumns extends ColumnsBase<ContentAudioRow> {
    static readonly columnsKey = 'Content.ContentAudio';
    static readonly Fields = fieldsProxy<ContentAudioColumns>();
}

File: GXpert/GXpert.Web/Modules/Content/ContentAudio/ContentAudioColumns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace GXpert.Content.Columns;

[ColumnsScript("Content.ContentAudio")]
[BasedOnRow(typeof(ContentAudioRow), CheckNames = true)]
public class ContentAudioColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int Id { get; set; }
    [EditLink]
    public string MediaFile { get; set; }
    public string HlsFile { get; set; }
    public string FileKeyUrl { get; set; }
    public string Iv { get; set; }
    public string ContentTitle { get; set; }
    public string LanguageName { get; set; }
    public string Username { get; set; }
   
}